queries/synths/useRedeemableDeprecatedSynthsQuery.ts

```diff
@@ -0,0 +1,76 @@
+import { ethers } from 'ethers';
+import { Provider, Contract } from 'ethcall';
+import { useQuery, UseQueryOptions } from 'react-query';
+import { CurrencyKey } from '@synthetixio/contracts-interface';
+import { wei } from '@synthetixio/wei';
+
+import { DeprecatedSynthBalance, DeprecatedSynthsBalances } from '@synthetixio/queries';
+import { getProxySynthSymbol } from './utils';
+import Connector from 'containers/Connector';
+import SynthRedeemerABI from 'lib/abis/SynthRedeemer.json';
+
+const ethCallProvider = new Provider();
+
+const useRedeemableDeprecatedSynthsQuery = (
+	walletAddress: string | null,
+	options?: UseQueryOptions<DeprecatedSynthsBalances>
+) => {
+	const { synthetixjs, network, provider } = Connector.useContainer();
+	return useQuery<DeprecatedSynthsBalances>(
+		['WalletBalances', 'RedeemableDeprecatedSynths', network.id, walletAddress],
+		async () => {
+			await ethCallProvider.init(provider as any);
+			const {
+				contracts: { SynthRedeemer },
+			} = synthetixjs!;
+
+			const synthDeprecatedFilter = SynthRedeemer.filters.SynthDeprecated();
+			const deprecatedSynthsEvents = await SynthRedeemer.queryFilter(synthDeprecatedFilter);
+			const deprecatedProxySynthsAddresses: string[] = deprecatedSynthsEvents.map(
+				(e: any) => e.args?.synth ?? ''
+			);
+
+			const Redeemer = new Contract(SynthRedeemer.address, SynthRedeemerABI);
+
+			const symbolCalls = [];
+			const balanceCalls = [];
+
+			for (const addr of deprecatedProxySynthsAddresses) {
+				symbolCalls.push(getProxySynthSymbol(addr));
```

Comment:
Instead of doing a call for the symbol, you can pull the synths from either: https://github.com/Synthetixio/js-monorepo/blob/master/packages/contracts-interface/src/index.ts#L147-L148 and I believe one of them is tied to an address